sources/packages/web/src/services/InstitutionService.ts

```diff
@@ -102,7 +104,9 @@ export class InstitutionService {
 
   public async updateInstitutionLocation(
     locationId: number,
-    institutionLocation: InstitutionLocationFormAPIInDTO,
+    institutionLocation:
+      | InstitutionLocationPrimaryContactAPIInDTO
+      | AESTInstitutionLocationAPIInDTO,
```

Comment:
promise?

Reply:
I would suggest to name the DTOs without client type added to it. In this case AEST can update all the properties of institution location. So it can be named as InstitutionLocationAPIInDTO instead of AESTInstitutionLocationAPIInDTO.